modules/govuk_mysql/templates/usr/local/bin/xtrabackup_s3_restore.erb

```diff
@@ -18,29 +19,42 @@ if test -f /var/lib/puppet/state/agent_disabled.lock; then
   exit 1
 else
 
-  govuk_puppet --disable "MySQL environment sync (by $0)"
+  /usr/local/bin/govuk_puppet --disable "MySQL environment sync (by $0)"
```

Comment:
Oooh in theory this could now use @deanwilson's new method to apply locks. I think the method would be something like `govuk_puppet -r 'MySQL environment sync' -u '$0' --disable`, but double check the script.